pkg/volume/iscsi/iscsi_util_test.go

```diff
@@ -69,3 +71,39 @@ func TestExtractPortalAndIqn(t *testing.T) {
 		t.Errorf("extractPortalAndIqn: got %v %s %s", err, portal, iqn)
 	}
 }
+
+func fakeOsStat(devicePath string) (fi os.FileInfo, err error) {
+	var cmd os.FileInfo
+	return cmd, nil
+}
+
+func fakeFilepathGlob(devicePath string) (globs []string, err error) {
+	return []string{devicePath}, nil
+}
+
+func TestWaitForPathToExist(t *testing.T) {
+	devicePath := []string{"/dev/disk/by-path/ip-127.0.0.1:3260-iqn.2014-12.com.example:test.tgt00-lun-0",
+		"/dev/disk/by-path/pci-0000:00:00.0-ip-127.0.0.1:3260-iqn.2014-12.com.example:test.tgt00-lun-0"}
+	osStat = fakeOsStat
+	defer func() { osStat = os.Stat }()
```

Comment:
hmm.. i think it would be cleaner if osStat and filepathGlob were passed as arguments to the function; analogous to NewBuilder and NewBuilderInternal.